scripts/pipelines/azure-devops/templates/library-package/library-package-pipeline.cfg

```
@@ -12,33 +12,73 @@ sourceBranch="feature/library-package-pipeline"
 pipelinePath=".pipelines"
 # Path to the scripts.
 scriptFilePath=".pipelines/scripts"
-#Feed name
-feed="maven-feed"
+
+# Determine feed name and type
+case "$language" in
+
+    "java")
+        feed="maven-feed"
+        feedType="Maven"
+    ;;
+
+    "node")
+        feed="npm-feed"
+        feedType="npm"
+    ;;
+
+    *)
+        echo "Language '${language}' is not supported."
+    ;;
+esac
 
 # Function that copies the script to build the application.
 function copyScript {
     # Copy the script.
     cp "${hangarPath}/${templatesPath}/${language}-${scriptFile}" "${localDirectory}/${scriptFilePath}/${scriptFile}"
 
-    #Get organization
+    # Get organization
     cd "${localDirectory}"
     organization=$(git config --get remote.origin.url | cut -d/ -f4)
 
-    #Create PAT
+    # Create PAT
     echo -e "${green}Creating a PAT with necessary permissions..."
     patToken=$(az rest --method post --uri "https://vssps.dev.azure.com/${organization}/_apis/tokens/pats?api-version=7.1-preview.1" --resource "https://management.core.windows.net/" --body '{ "displayName": "libraryDeployToken", "validTo": "2099-02-25T11:44:36.1966667Z", "scope": "vso.packaging_manage" }')
     patToken=$(echo "$patToken" | python -c  "import sys, json; print(json.load(sys.stdin)['patToken']['token'])")
 
-    #Create feed
-    echo -e "${green}Creating a Maven feed..."
+    # Create feed
+    echo -e "${green}Creating a ${feedType} feed..."
```

```suggestion
    echo -e "${green}Creating a ${feedType} feed...${white}"
```

Without changing back to white all the next lines are green